### PR TITLE
Add create database endpoint.

### PR DESCRIPTION
**Pull request by H0R15H0**
Hello, there!
NotionAPI now has an endpoint to [create a database](https://developers.notion.com/changelog/create-new-databases-with-post-databases), so I've implemented it.
There may be some poor English, so if you could check it, that would be great!

### CODE REVIEW

File: lib/notion/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 module Notion
   VERSION = '0.0.7'
-  NOTION_REQUEST_VERSION = '2021-05-11'
+  NOTION_REQUEST_VERSION = '2021-05-13'
```
Comment (phacks) [inline lib/notion/version.rb:4]: Moving from `05-11` to `05-13` is a breaking change in Notion API, according to this piece of docs: https://developers.notion.com/reference/versioning (`text` becomes `rich_text`)

The breaking change does not appear to affect any of the gem code as far as I can tell, but some of the docs might then be outdated (in the [Create a Page endpoint](https://github.com/orbit-love/notion-ruby-client#pages) and the [Create child blocks endpoint](https://github.com/orbit-love/notion-ruby-client#blocks).

Would you be willing to try those endpoints out with `text` replaced by `rich_text` and see if they still work? If so, could you update those bits in the docs by replacing the property name?

**Review by phacks — COMMENTED** [review]
Thank you so much @H0R15H0 for the PR—as with any new Notion API capability, this one opens new use cases and we’re excited to see what folks will use it for.

I added a small comment for a potential breaking change in the docs (needed to use this new endpoint), otherwise the PR looks great and I appreciate the addition of specs.

File: lib/notion/version.rb (hunk 1 above)
Comment (H0R15H0) [reply lib/notion/version.rb:4]: Thank you for your review @phacks!

I'm sorry for overlooking the change. 
It seems that the change only affects `Property value object` in page object (not affects `blocks`).

> Would you be willing to try those endpoints out with text replaced by rich_text and see if they still work? If so, could you update those bits in the docs by replacing the property name?

I've made this replacement and confirmed `Create a Page endpoint` works! So I'm thinking of updating the docs as well! 
Would it be better to split the PR on this update?

I'm sorry for my poor English, but I hope you can understand.

Comment (phacks) [reply lib/notion/version.rb:4]: Your English is very good! As a non-native speaker myself I wouldn’t have known if you hadn’t told me 🙂

Bundling both changes in this very PR seems easier, so let’s do this!

Comment (H0R15H0) [reply lib/notion/version.rb:4]: Thank you for your kind words!
I just pushed the changes, so could you check it?

**Review by phacks — APPROVED** [review]
Congratulation landing this PR @H0R15H0! Thanks again for your contribution 🙌

**Pull request merged**